stable/mssql-linux/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: SQL Server 2017 Linux Helm Chart
 name: mssql-linux
-version: 0.5.0
+version: 0.6.0
```

Comment:
Because this is being added in a backward compatible way you can go with version `0.5.1`. Prior to a 1.0.0 release the minor incrementing means a break in backwards compatibility. No breaking change happened here.

Reply:
Oh cool, I didn't know that! Will fix :)